scenarios/scenario-1/wso2-apim.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: cluster-1
 spec:
   pattern: Pattern-1
+  expose: ""
```

Comment:
Why this is defined here? It is not required and it is an optional field.

Reply:
Fixed with c4be73d